web/html/src/components/table/TableDataHandler.tsx

```diff
@@ -320,18 +324,13 @@ export class TableDataHandler extends React.Component<Props, State> {
 
     const handleSearchPanelSelectAll = () => {
       this.setState({ loading: true }, () => {
-        this.state.provider.getIds(
-          promise =>
-            promise
-              .then(data => {
-                const selected = selectedItems;
-                this.setSelection(selected.concat(data.filter(id => !selected.includes(id))));
-              })
-              .finally(() => this.setState({ loading: false })),
-          this.state.criteria
-        );
-      });
-    };
+        const data = selectEnabledItems.map(item => this.props.identifier(item));
+        const selected = selectedItems;
+        this.setSelection(selected.concat(data.filter(id => !selected.includes(id))));
+      })
+      this.setState({ loading: false }),
+        this.state.criteria
+    }
```

Comment:
Removing that piece of code is not a good idea: the filtered select all wouldn't work with asynchronous data providers anymore. I'll try to come up with a solution